modules/signing_worker/files/requirements.txt

```diff
@@ -0,0 +1,5 @@
+-e git+https://github.com/escapewindow/scriptworker-scripts#egg=mozbuild&subdirectory=vendored/mozbuild
+-e git+https://github.com/escapewindow/scriptworker-scripts#egg=scriptworker_client&subdirectory=scriptworker_client
+-e git+https://github.com/escapewindow/scriptworker-scripts#egg=iscript&subdirectory=iscript
+scriptworker
+# TODO add widevine once we have a wheel or private repo solution
```

Comment:
Until we have some solution for widevine (github secret for cloning a private repo?) we'll still have at least one manual step before we can go from imaged/puppetized to live.

Reply:
If there's a manual step I should remove the `launchctl load` directive and have someone do that manually.